backend/init_data/skills/sandbox/_base.py

```
@@ -0,0 +1,501 @@
+            workspace_ref="my-repo"
+        )
+        if error:
+            # Handle error
+            pass
+        else:
+            # Use sandbox
+            pass
+    """
+
+    # Class-level dictionary to store singleton instances per task_id
+    _instances: dict[int, "SandboxManager"] = {}
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)